catppuccin-theme.el

```diff
@@ -30,846 +32,934 @@ The theme has to be reloaded after changing anything in this group."
+               (tab-line :inherit tab-bar)
+               (tab-line-tab :inherit tab-bar-tab)
+               (tab-line-tab-inactive :inherit tab-bar-tab-inactive)
                (tab-line-tab-current :inherit tab-line-tab)
-               (tab-line-close-highlight :foreground ,catppuccin-red)
-               ;; telephone-line
-               (telephone-line-accent-active :background ,catppuccin-bg :foreground ,catppuccin-pink)
-               (telephone-line-accent-inactive :background ,bg2 :foreground ,catppuccin-purple)
-               (telephone-line-unimportant :background ,catppuccin-bg :foreground ,catppuccin-comment)
+               (tab-line-highlight :background ,ctp-surface1)
                ;; term
-               (term :foreground ,catppuccin-fg :background ,catppuccin-bg)
-               (term-color-black :foreground ,catppuccin-bg :background ,catppuccin-comment)
-               (term-color-blue :foreground ,catppuccin-purple :background ,catppuccin-purple)
-               (term-color-cyan :foreground ,catppuccin-cyan :background ,catppuccin-cyan)
-               (term-color-green :foreground ,catppuccin-green :background ,catppuccin-green)
-               (term-color-magenta :foreground ,catppuccin-pink :background ,catppuccin-pink)
-               (term-color-red :foreground ,catppuccin-red :background ,catppuccin-red)
-               (term-color-white :foreground ,catppuccin-fg :background ,catppuccin-fg)
-               (term-color-yellow :foreground ,catppuccin-yellow :background ,catppuccin-yellow)
+               (term :foreground ,ctp-text :background ,ctp-base)
+               (term-color-black :foreground ,ctp-base :background ,ctp-base)
+               (term-color-bright-black :foreground ,ctp-surface1 :background ,ctp-surface1)
+               (term-color-red :foreground ,ctp-red :background ,ctp-red)
+               (term-color-bright-red :foreground ,ctp-maroon :background ,ctp-maroon)
+               (term-color-green :foreground ,ctp-green :background ,ctp-green)
+               (term-color-bright-green :foreground ,ctp-green :background ,ctp-green)
+               (term-color-yellow :foreground ,ctp-peach :background ,ctp-peach)
+               (term-color-bright-yellow :foreground ,ctp-yellow :background ,ctp-yellow)
+               (term-color-blue :foreground ,ctp-blue :background ,ctp-blue)
+               (term-color-bright-blue :foreground ,ctp-sapphire :background ,ctp-sapphire)
+               (term-color-magenta :foreground ,ctp-mauve :background ,ctp-mauve)
+               (term-color-bright-magenta :foreground ,ctp-lavender :background ,ctp-lavender)
+               (term-color-cyan :foreground ,ctp-sky :background ,ctp-sky)
+               (term-color-bright-cyan :foreground ,ctp-teal :background ,ctp-teal)
+               (term-color-white :foreground ,ctp-text :background ,ctp-text)
+               (term-color-bright-white :foreground ,ctp-overlay2 :background ,ctp-overlay2)
```

Comment:
This should probably follow [the terminal style guide](https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#terminals), handling Latte differently.